quickstart.md

```diff
@@ -931,6 +931,9 @@ Confidential Containers integrates several components. If you run into problems,
 it can sometimes be difficult to figure out what is going on or how to move forward.
 Here are some tips.
 
+If you get stuck or find a bug, please make an issue on this repository or
+the repository for the component in question.
```

Comment:
@wainersm how's about this? (original idea was if people didn't know where to go point them at one central spot with many eyes.)

```suggestion
the repository for the component in question, e.g., [the operator](https://github.com/confidential-containers/operator/issues).
```

Reply:
It sounds good @dcmiddle !

Reply:
updated